### DISCUSSION TITLE
Version identity issue and notifications

### DISCUSSION THREAD

**Discussion by KevQuoi** [WindowTop/WindowTop-App#73] [General]
I've downloaded and updated to version 5.1.9 however the app is still reporting the version to be 5.1.8 and thus wants to update!

Could you please advise?

Second note, is there a way to disable the notifications when actions are taken? 

Ta

**Comment by gileli121** [comment]
@KevQuoi 
It already reported and fixed here
https://github.com/gileli121/WindowTop/issues/70
Download again. It will be now 5.1.9

About your suggestion to disable notifications,
I will remove the notifications in next version or add option in the settings to disable them

**Reply by KevQuoi** [reply]
Fantastic, many thanks for the speedy fix. Keep up the great work :-)
